sktime/utils/validation/forecasting.py

```diff
@@ -127,7 +130,8 @@ def check_y(y, allow_empty=False, allow_constant=True, enforce_index_type=None):
         y,
         enforce_univariate=True,
         allow_empty=allow_empty,
-        allow_numpy=False,
+        # allows numpy
+        allow_numpy=True,
```

Comment:
is this the only non-docstring related change?
In any case, the PR preamble should make reference to the docstring fixes addressing the new docstring checks.